image/Makefile

```diff
@@ -0,0 +1,29 @@
+SHELL:=/bin/bash
+ENGINE_DIR:=$(CURDIR)/../../engine
+CLI_DIR:=$(CURDIR)/../../cli
+VERSION?=0.0.0-dev
+STATIC_VERSION=$(shell ../static/gen-static-ver $(ENGINE_DIR) $(VERSION))
+DOCKER_HUB_ORG?=docker
+ENGINE_IMAGE?=ce-engine
+
+.PHONY: help
+help: ## show make targets
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {sub("\\\\n",sprintf("\n%22c"," "), $$2);printf " \033[36m%-20s\033[0m  %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+.PHONY: clean
+clean: ## remove build artifacts
+	docker rmi $(DOCKER_HUB_ORG)/$(ENGINE_IMAGE):$(STATIC_VERSION)
+
+.PHONY: static
+static: static-linux
+
+.PHONY: static-linux
+static-linux:
```

Comment:
Oh after a second look we can probably just rename these static related targets to just `image`

Reply:
At first I was keeping them just in case we need to make OS specific binaries, but that'll probably require a more extensive refactoring to achieve, and it's just theoretical at this point, so renaming seems fine to me.

Reply:
I've renamed it to `image-linux`